### DISCUSSION TITLE
canAccessPanel()

### DISCUSSION THREAD

**Discussion by brunoalves21** [filamentphp/filament#8372] [Help]
Call to undefined method App\Models\User::canAccessPanel() 

I recently run a v3.0 project and everything is good without the canAccessPanel() function, but now, it is required. Is that normal? 

I confirm the versions and I got this:

filament/filament
  "version": "v3.0.19", - work
  "version": "v3.0.42", - dont work

**Comment by leandrocfe** [comment]
Did you do this?
```php
//app/Models/User.php
use Filament\Models\Contracts\FilamentUser;
class User extends Authenticatable implements FilamentUser
...
```

**Comment by danharrin** [comment]
Ohh, I know the problem here. I can fix it in the next release

**Reply by brunoalves21** [reply]
Thanks dan :) 

But let me know. Still need to add the method and "implements FilamentUser" or will it by default let all the user to access? Like we can see on the docs 👍🏽

**Reply by danharrin** [reply]
`FilamentUser` needs to be there for production, but it is not required locally. Anyway, it should be fixed in the latest release

**Reply by brunoalves21** [reply]
Got it Dan. Much thanks 😃
